src/controller/getDreamControllerById.js

```diff
@@ -0,0 +1,47 @@
+import Questionnaire from '../models/dream';
+const mongoose = require('mongoose');
+
+function isNumber(posibleId){
+	let id = parseInt(posibleId, 10);
```

Comment:
You are using parseInt two times over the same id (once in idVerification() and again here)